src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -47,6 +53,15 @@ struct event_entry_t {
     } else {
       creation_time = ceph::coarse_real_clock::zero();
     }
+    if (struct_v > 2) {
+      decode(time_to_live, bl);
+      decode(max_retries, bl);
+      decode(retry_sleep_duration, bl);
+    } else {
+      time_to_live = UINT32_MAX;
```

Comment:
please create a special constexpr for "global override" so it is clear what these values are.
als, you can set it in lines 28-30, so the "else" here is not needed